src/phases/execution/ClusteredCollection.yml

```diff
@@ -134,17 +133,86 @@ Actors:
         Operations:
         - OperationName: find
           OperationCommand:
-            Filter: {a: {^FastRandomString: {length: 6, alphabet: "0123456789"}}}
+            Filter: {a: *RandomSecondaryKey}
+
+# Phase 6: Range lookup with both primary and secondary key. Primary key predicate is more selective.
+- Name: RangeLookupBySelectivePrimaryAndWideSecondaryIndexes
+  Type: CrudActor
+  Database: *Database
+  Threads: 16
+  Phases:
+    OnlyActiveInPhases:
+      Active: [6]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Duration: 1 minute
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {
+              _id: {^Choose: {
+                from: [
+                  {$gte: &BigId {^Join: {array: [
```

Comment:
Minor: We can move these values up instead of cluttered in the middle of workload.